CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.3.2
+
+- version() method - A new version has been introduced in the respective SDKs which will let the developers know the version of Phyllo SDK integrated in their app.
```

Comment:
Put this changelog

- Added `version` method to help with debugging.
- More robust internal logging and debugging mechanisms.

Reply:
@akshaybhalotia ok sure , let me change it

Reply:
@akshaybhalotia updated this changes